### ISSUE TITLE
win10 AH01630: client denied by server configuration for .tmb folder

### ISSUE THREAD

**Issue by stalker780** [Studio-42/elFinder#3047]
Hi,

Does anybody here run elFinder with windows 10 build 1909 + Apache 2.4?

I have a strange problem, and suspect it happened after updating windows 10 to build 1909.

Apache cannot access hidden .tmb folder saying AH01630: client denied by server configuration, http code 403.
Same with Apache+nginx, http code 404.

If I change elFinder config tmbPath and tmbURL from .tmb to _tmb, it works like a charm. All images from _tmb start loading.

I did not change Apache config for about a year. And an only thing I think of is update of Windows :(

I understand this is not elFinder bug, just asking if someone else have faced this poroblem.
If it is a new windows bug/feature not sure Microsoft is going to fix this soon. So many developers can run into it.

**Comment by nao-pon** [comment]
I tested elFinder 2.1.50 on Windows 10 (1909) - Apache/2.4.41 (Win64) OpenSSL/1.1.1c PHP/7.3.10.

That was no problems. Please check the server settings again.

**Comment by stalker780** [comment]
> I tested elFinder 2.1.50 on Windows 10 (1909) - Apache/2.4.41 (Win64) OpenSSL/1.1.1c PHP/7.3.10.
> 
> That was no problems. Please check the server settings again.

Thank you for this information.

I have also reverted a Windows backup to 1903 on my notebook and hidden folders are working again. Apache and nginx configs and versions are absolutely the same on my main PC running 1909 and notebook running 1903.

Also if it was a config problem, it won't influence both Apache and nginx. Both of them stopped working with hidden folders at the same time.

So it definitely was broken by windows update. I still don't understand how to fix this. I used these configs for years since winxp, and there never was any problem.
